test/e2e/framework/auth/helpers.go

```diff
@@ -57,12 +64,14 @@ func WaitForNamedAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviews
 			User: user,
 		},
 	}
+
 	err := wait.Poll(policyCachePollInterval, policyCachePollTimeout, func() (bool, error) {
 		response, err := c.SubjectAccessReviews().Create(review)
 		// GKE doesn't enable the SAR endpoint.  Without this endpoint, we cannot determine if the policy engine
 		// has adjusted as expected.  In this case, simply wait one second and hope it's up to date
+		// FIXME(johnschnake): Should have a check for the provider here but that introduces too tight of
+		// coupling with the `framework` package.
 		if apierrors.IsNotFound(err) {
-			klog.Info("SubjectAccessReview endpoint is missing")
```

Comment:
I ended up including some logging functionality in the IsRBACEnabled method; if you want me to add this back I can.

It is not an error condition based for tests and seems like it only happens in known situations that won't surprise anyone.

Reply:
I think we should leave the log line for now using logf below